### DISCUSSION TITLE
GitHub support MIA - GitHub actions is disabled for my repository<title>

### DISCUSSION THREAD

**Discussion by halih** [community/community#48767] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I have a ticket (#2015091) open for 13 days at this point with no response.

**Comment by ettaboyle** [comment]
Thank you for your feedback and your patience, @halih I can see that your ticket is still open and awaiting triage from our team. 

If the issue is not confidential or account specific I encourage you to look for a similar question in our discussions or post a new post. Whenever possible we like to help users with issues that others might also face so that more information is available for others. 

Cheers!
